src/Spatial/Euclidean/Circle2D.cs

```diff
@@ -131,6 +134,63 @@ public static Circle2D FromPoints(Point2D pointA, Point2D pointB, Point2D pointC
             return new Circle2D(center, radius);            
         }
 
+        /// <summary>
+        /// Returns intersection a point2D array if this circle and the given line have the intersections
```

Comment:
I think this comment could be improved a bit.

Reply:
I was thinking something like "Returns the intersections of this circle with the given line."

Reply:
Much better. Simple, precise and clean.
Thank you.

Reply:
But still not changed in the PR 😄